README.md

```diff
@@ -97,8 +97,23 @@ Push the changes to a branch instead of publishing to the NPM registry.
 
 Custom message to added to the beginning of the release GitHub comment.
 
+**`is_global` (optional)**
+
+If true, the generated GitHub comment will show the npm command to install your packages globally.
```

Comment:
```suggestion
If true, the generated GitHub comment will show the command to install your packages globally.
```

Reply:
One thing I am not super happy with is that snapit works for `npm` `pnpm` `yarn` and whatever else could arrive. How could these changes be more package agnostic?